app/src/main/java/com/example/b07group7project/FragmentSwitch.java

```diff
@@ -3,5 +3,5 @@
 import androidx.fragment.app.Fragment;
 
 public interface FragmentSwitch {
-    void replaceFragment(Fragment fragment);
```

Comment:
It would lead to less changes if you keep both methods, have void replaceFragment(Fragment) in the super implementation just redirect to replaceFragment(Fragment, false). Changing this means I need to change tests and a bunch of abstractions to make the testing easier

Reply:
idk if i can do this since FragmentSwitch is an Interface
The only other option i can think of is having to implement the redirect in each activity unless you had another idea